### PR TITLE
Enhance spec of lvs_noheadings with seg_monitor option

### PR DESCRIPTION
**Pull request by JoySnow**

### CODE REVIEW

**Comment by JoySnow** [comment]
https://github.com/RedHatInsights/insights-frontend-assets/pull/188

**Comment by JoySnow** [comment]
Related： https://github.com/RedHatInsights/insights-frontend-assets/pull/152

**Comment by JoySnow** [comment]
Q:  Why "Change the spec of lvs_noheadings , but not lvs_noheadings_all" ?

A:  
  "lvs_noheadings" & "lvs_noheadings_all" are for different usage.   
   As we had discussed when adding for https://github.com/RedHatInsights/insights-core/pull/1331,  add more option to "lvs_noheadings" will lead to more resource consuming when collecting data.  
   Hence, talked with @xiangce , we think it would be better to not add the spec to "lvs_noheadings_all" this time.

**Comment by xiangce** [comment]
test this please

**Comment by JoySnow** [comment]
Spec changes PR move to https://github.com/RedHatInsights/insights-core-assets/pull/31

File: insights/parsers/lvm.py [hunk 1]
```diff
@@ -512,7 +512,8 @@ class Lvs(Lvm):
         "LVM2_CACHE_READ_MISSES": "CacheReadMisses",
         "LVM2_LV_DESCENDANTS": "Descendants",
         "LVM2_REGION_SIZE": "Region",
-        "LVM2_SEGTYPE": "SegType"
+        "LVM2_SEGTYPE": "SegType",
+        "LVM2_SEG_MONITOR": "Monitor"
```
Comment (bfahr) [inline insights/parsers/lvm.py]: @JoySnow can you please modify the tests to include testing for this additional column?

Comment (JoySnow) [reply insights/parsers/lvm.py]: @bfahr I had modified the tests.

Comment (xiangce) [reply insights/parsers/lvm.py]: @bfahr has confirmed this modification

**Pull request merged**